src/core/Item.h

```diff
@@ -124,6 +124,13 @@ class CItem
     LAST_FIELD
   };
 
+  static bool IsTimeField(int ft)
```

Comment:
Although the _FieldType_ data type exists as an enumeration, the API uses an integer type instead. Is this for historical reasons, programming reasons, or is a C API intended to be provided to make it easier for other projects to create bindings?

Reply:
Historical reasons. Today I'd keep this as a reminder that the values passed may be outside those defined in the enum, as in forward compatibility. That is, reading a database written on a newer version that has fields (enums) that are unrecognized by the reading version. PasswordSafe will read and write these fields, but won't know how to display them.